### PR TITLE
readme updated

### LINKED ISSUE #4: Update readme based on backend repo
https://github.com/MultiEmail/MultiEmail-backend/edit/main/readme.md

### PR DESCRIPTION
**Pull request by samiulbasirfahim**
### Description

Updated readme based on backend repo

### Checklist:

- [ ] This pull request follow our contribution guidelines
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (postman and swagger)
- [ ] I have written tests for the code
- [ ] I have updated .env.example file for new .env variables

### What is current behavior?

Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,50 @@
...
-## Future plans
-
-- Create mobile with same features
+# Technologies Used
+
+- ExpressJs
+- Mongoose
+- Typegoose
+- TypeScript
+- Passport
```
Comment (aayushchugh) [inline README.md:7]: You have to update this tech stack make it

- React
- Redux
- Tailwind CSS

Comment (aayushchugh) [reply README.md]: I have created commit for solving this

**Review by aayushchugh — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,19 +1,48 @@
...
+- send emails
+- recive emails
+- Connections through other parties ie discord, twitter, facebook etc..
+
+# Setup
+
+## Windows
+
+```
+git clone https://github.com/MultiEmail/MultiEmail-backend.git
```
Comment (KanLSK) [inline README.md]: Isn’t this should be the frontend instead?

File: README.md [hunk 3]
```diff
@@ -1,19 +1,48 @@
...
+cd MultiEmail-backend
```
Comment (KanLSK) [inline README.md]: and also this -> “cd MultiEmail-frontend”

File: README.md [hunk 4]
```diff
@@ -1,19 +1,48 @@
...
+yarn install
+yarn dev
+```
+
+# Linux
+
+```
+git clone https://github.com/MultiEmail/MultiEmail-backend.git && cd MultiEmail-backend && yarn install && yarn dev
```
Comment (KanLSK) [inline README.md]: Also this “ git clone https://github.com/MultiEmail/MultiEmail-frontend.git && cd MultiEmail-frontend && yarn install && yarn dev

**Review by KanLSK — APPROVED** [review]

**Review by Karol-Ari — APPROVED** [review]

**Pull request merged**